KthLastElement.java

```diff
@@ -0,0 +1,46 @@
+package com.katermar;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Created by USER on 5/15/2017.
+ */
+public class KthLastElement {
+
+    private Integer value;
+    public KthLastElement next;
+
+    public KthLastElement(Integer value) {
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return "KthLastElement{" +
+                "value=" + value +
+                '}';
+    }
+
+    public KthLastElement findKth(Integer k, KthLastElement head) {
+        KthLastElement current = head, before = head;
+
+        if (k < 1 || current == null) {

```

Comment:
This is not exactly correct. k == 0 should result in last element of the list being returned rather than first.